docs/source/installation/installation.txt

```diff
@@ -101,11 +179,47 @@ Load some defaults.
         python manage.py load_npo_defaults
 
 
-To create an administrator login, run the following command and fill in the prompts:
+To create an adminstrator login, run the following command and fill in the prompts:
 ::
 
     python manage.py createsuperuser
 
+Finally, we can use the runserver command so that we can view the site in our browser:
+::
+
+    python manage.py runserver
+
+Open http://127.0.0.1:8000/ in your browser to see your tendenci site!
+
+
+Theming Options
+----------------
+
+**Optional:** If you want to use a different theme, you can set that theme with the following command, replacing `my-theme-name` with the name of your theme:
+::
+
+    python manage.py set_theme my-theme-name
+
+
+Scalable Installation
+===================
+
+With a working test setup you can upgrade it to a fully functional, scalable install.
+Below are some instructions to create a Tendenci site on an Ubuntu server (14.04 is recommended).
+
+All of the commands below should be run on the command line after you are
+connected to the server and build on the work done in the previous sections.
+
+Estra packages
```

Comment:
I just noticed this Typo; if there is more work requested for this branch i'll fix it then.